deploy/helm/sumologic/templates/logs/collector/otelcol-cloudwatch/statefulset.yaml

```diff
@@ -0,0 +1,119 @@
+{{- if .Values.sumologic.logs.collector.otelcloudwatch.enabled }}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ template "sumologic.metadata.name.logs.collector.statefulset" . }}
+  labels:
+    app: {{ template "sumologic.labels.app.logs.collector.statefulset" . }}
+    {{- include "sumologic.labels.common" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "sumologic.labels.app.logs.collector.statefulset" . }}
+  serviceName: {{ template "sumologic.labels.app.logs.cloudwatch.service-headless" . }}
+  template:
+    metadata:
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/logs/collector/otelcol/configmap.yaml") . | sha256sum }}
```

Comment:
Please do not reuse collector configuration for fargate usecase

Reply:
Added a separate configuration file for the fargate usecase